diff --git a/jw_player.info b/jw_player.info
old mode 100755
new mode 100644
index d0313d8..a12f33c
--- a/jw_player.info
+++ b/jw_player.info
@@ -5,4 +5,4 @@ core = 7.x
 dependencies[] = file
 dependencies[] = libraries
 dependencies[] = ctools
-files[] = jw_player.module
\ No newline at end of file
+configure = admin/config/media/jw_player/settings
diff --git a/jw_player.install b/jw_player.install
old mode 100755
new mode 100644
diff --git a/jw_player.make.example b/jw_player.make.example
index 9dfbf85..751ed0e 100644
--- a/jw_player.make.example
+++ b/jw_player.make.example
@@ -11,4 +11,4 @@ libraries[jwplayer_skin_modieus][directory_name] = "jwplayer_skins/modieus"
 
 libraries[jwplayer_skin_snel][download][type] = "get"
 libraries[jwplayer_skin_snel][download][url] = "http://www.longtailvideo.com/files/skins/snel/5/snel.zip"
-libraries[jwplayer_skin_snel][directory_name] = "jwplayer_skins/snel"
\ No newline at end of file
+libraries[jwplayer_skin_snel][directory_name] = "jwplayer_skins/snel"
diff --git a/jw_player.module b/jw_player.module
old mode 100755
new mode 100644
diff --git a/plugins/export_ui/jw_player_ctools_export_ui.inc b/plugins/export_ui/jw_player_ctools_export_ui.inc
old mode 100755
new mode 100644
